### DISCUSSION TITLE
My college doesn't provide me with an university associated mail ID for students but it's still asking for one

### DISCUSSION THREAD

**Discussion by abhinavkothakota** [community/community#145338] [GitHub Education]
### Hi everyone,

Please [add and verify your school-issued email address ](https://github.com/settings/emails)in your account settings – or a contact email if you do not have one. Once your email is verified, refresh this page, I also added one of my other mail ids as a contact mail id and verified, still doesnt work

**Comment by alex-frost-dev** [comment]
Same here...
